README.rst

```diff
@@ -571,9 +584,10 @@ do the following:
            hooks:
            -   id: darker
 
-4. install the Git hook scripts::
+4. install the Git hook scripts and update to the newest version::
 
        pre-commit install
+       pre-commit autoupdate
```

Comment:
Seems that with `pre-commit` we can't use "compatible release" version specifiers, so it will happily upgrade across major version updates as well. Could a `pre-commit` expert confirm?

Reply:
You can use `--freeze` https://pre-commit.com/#pre-commit-autoupdate

Reply:
Thanks, nice to learn about those options. However, I guess neither one adds substantial value in this case, and it's best to keep the example simple as is, would you agree?